### DISCUSSION TITLE
I feel like configparser can be removed from setup.py and requirements.txt

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#118] [Q&A] [answered]
I feel like ``configparser`` can be removed from setup.py and requirements.txt

I feel like there is no place to use it.

**Comment by jertel** [comment, marked as answer]
Perhaps the early days of ElastAlert used a different config file format. If that's the case then it does make sense to remove this.

**Reply by nsano-rururu** [reply]
I am planning to make a pull request to delete configparser.
